### PR TITLE
Updated contact form

### PR DESCRIPTION
**Pull request by abeltsew**
This is A second pull because of accidental approval and merge of the project on student dashboard.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -216,5 +216,82 @@ <h2 class="aboutTitle">Skills</h2>
...
+          class="fieldSeparator"
+          src="/images/separatorForm.svg"
+          alt="separator"
+        />
+        <input
+          id="email"
+          name="email"
+          type="email"
+          placeholder="Email address"
+        />
```
Comment (AdnanOlarmmi) [inline index.html]: - [x] According to the requirements, all the `input` fields should be `required` 😀, I suggest that you implement these changes. 🙂.

File: index.html [hunk 2]
```diff
@@ -216,5 +216,82 @@ <h2 class="aboutTitle">Skills</h2>
...
+          class="fieldSeparator"
+          src="/images/separatorForm.svg"
+          alt="separator"
+        />
+
+        <textarea
+          id="message"
+          name="message"
+          placeholder="Enter Text Message"
+        ></textarea>
```
Comment (AdnanOlarmmi) [inline index.html]: - [x] Kindly, update the `textarea` tag to take in a maximum of 500 characters, by making use of the `maxlength` attribute of the `input`s/`textarea` tags, as suggested in the requirement. 😀

File: index.html [hunk 3]
```diff
@@ -216,5 +216,82 @@ <h2 class="aboutTitle">Skills</h2>
...
+        action="https://formspree.io/f/mknarzzq"
+        method="post"
+      >
+        <input
+          id="fullName"
+          name="fullName"
+          type="text"
+          placeholder="Full Name"
+          required
+        />
```
Comment (AdnanOlarmmi) [inline index.html]: - [x] Kindly, update the name's `input` tag to take in a maximum of 30 characters, by making use of the `maxlength` attribute of the `input`s/`textarea` tags, as suggested in the requirement. 😀

File: style.css [hunk 4]
```diff
@@ -294,7 +294,6 @@ button:hover {
```
Comment (AdnanOlarmmi) [inline style.css:295]: - [x] [OPTIONAL] Kindly make sure that your design doesn't overflow, I suggest that you do it by setting the `overflow-x` property to `hidden`, you might choose any other way out 😀.

mobile

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @abeltsew and @abel-tefera, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Professional Readme.
- Nice UI design.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by souad988 — APPROVED** [review]
Hi @abeltsew , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔design well implemented.

## Optional suggestions
_Good job on applying changes requested by previous reviewer 👍, and Great work so far 💪
One last thing to work on is using the same contact form button color and textarea placeholder font color used in figma design don't forget to make these small changes to make your project look exactly like the provided design 😊

contact form tomp2

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**